api/app/tests/apps/api/v1/test_private_departments.py

```diff
@@ -189,7 +189,6 @@ def test_SIG_2287(self):
         category_assignment = data['categories'][0]
         category = category_assignment['category']
         category_url = category['_links']['self']['href']
-        self.assertEqual(expected_parent_url, category_url)
```

Comment:
should this be here, or at least I have seen it in a few outstanding PRs

Reply:
Yes it was a mistake I made when merging changes into this branch